try:
    result=35/0
except:
    print('errro happend')
finally:
    print('finally here')
print('done')